training/distributed_training/pytorch/data_parallel/bert/pytorch_smdataparallel_bert_demo.ipynb

```diff
@@ -22,7 +22,7 @@
     "6. Define training metrics\n",
```

Comment:
<p><code>build_and_push.sh</code> needs to be updated to include the command for logging the public ECR that host pytorch training image</p><pre class="ql-syntax" spellcheck="false">aws ecr get-login-password --region ${region} | docker login --username AWS --password-stdin 763104351884.dkr.ecr.${region}.amazonaws.com
</pre><p>otherwise users cannot pull from that account.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>